.gitignore

```diff
@@ -54,3 +54,4 @@ examples/imitation_learning_openbot/videos
 python/spear_sim.egg-info
 
 tools/scene_downloader/tmp
+tools/tmp
```

Comment:
I think we should also add `tools/sign_executable/tmp`, since we are creating this temporary folder in our `sign_executable.py` script.

Reply:
Agreed, although `sign_executable` now defaults to using `tools/tmp` as its `temp_dir`, because I wanted it to match the default `temp_dir` used by `build_executable`. But there is no harm in adding `tools/sign_executable/tmp` to the ignore list, and doing so would be consistent with `scene_downloader`.